src/Data/Vec/Base.agda

```diff
@@ -17,7 +17,7 @@ open import Data.These.Base as These using (These; this; that; these)
 open import Function.Base using (const; _∘′_; id; _∘_)
 open import Level using (Level)
 open import Relation.Binary.PropositionalEquality.Core using (_≡_; refl; trans; cong)
-open import Relation.Nullary.Decidable.Core using (does)
+open import Relation.Nullary.Decidable.Core using (does; T?)
```

Comment:
Actually, no need for this import of `T?` here: it's not used anywhere!

Reply:
Used in `countᵇ p = count (T? ∘ p)`

Reply:
🤦